data/block_export_data.go

```diff
@@ -20,6 +20,7 @@ type RoundProvider interface {
 type InitProvider interface {
 	Genesis() *bookkeeping.Genesis
 	NextDBRound() basics.Round
+	AdvanceDBRound()
```

Comment:
Where is this getting called?

Reply:
This would be called at the end of the pipeline so that the init provider would know when to increase the database round we are working on.  Right now the NextDBRound only gets the current round (maybe we should prefix it with "Get"), so we would need something to advance it.

Reply:
I definitely don't think we should have a public method for this. If I was a plugin developer it might be tempting to call it in order to skip ahead.